kpatch-build/create-klp-module.c

```diff
@@ -74,6 +74,8 @@ static struct symbol *find_or_add_ksym_to_symbols(struct kpatch_elf *kelf,
 	snprintf(pos, 32, "%lu", ksym->pos);
 	/* .klp.sym.objname.name,pos */
 	snprintf(buf, 256, KLP_SYM_PREFIX "%s.%s,%s", objname, name, pos);
+	free(name);
+	free(objname);
```

Comment:
There's no need for the original strdup's in the first place.  e.g.
```
objname = strdup(strings + rela->addend);
```
can just be simplified to
```
objname = strings + rela->addend;
```
And the same comment applies for the other changes.

Reply:
Yet, it's good ideas, it make the code more simple. Thank you for your review.